tests/ZendTest/Validator/DateTest.php

```diff
@@ -70,8 +70,10 @@ public function datesDataProvider()
             // int
             array(0,                         null,              true),
             array(1340677235,                null,              true),
-            // Commenting out, as value appears to vary based on OS
-            // array(999999999999,              null,              true),
+            // 32bit version of php will convert this to double
```

Comment:
I read this entirely wrong; this test **un**comments a test set. We're covered, and verified this passes.